docs/Deployment.md

```diff
@@ -1,6 +1,44 @@
-# Deployment Process
+# Deployment
 
-## Travis CI
+## Deployment Process
+
+### SSL
+
+Before deploying for the first time, you must set up SSL and configuration for
+Nginx, which we are using as a proxy server. This can be done by running the
+[`setup_deploy.sh`](../scripts/setup_deploy.sh) script. This runs the official
```

Comment:
This does not work as you think it should (see the built docs [here][built-docs]). The links are valid for within GitHub only, but invalid once built at RTD. Consider a hard link or removing them??? Same with the ones below.

[built-docs]: https://rocket2.readthedocs.io/en/deployment/docs/Deployment.html

Reply:
Sad. I've just gone ahead and removed them - I think it's obvious enough where they come from.